### DISCUSSION TITLE
Setting up discussions ends with Page not found error

### DISCUSSION THREAD

**Discussion by danielkcz** [community/community#22141] [New to GitHub]
<p>Hi, I am trying to set up new discussions in the repo <a href="https://github.com/mobxjs/mobx-react-lite" rel="nofollow noopener">https://github.com/mobxjs/mobx-react-lite</a>. Upon the click on “Set up discussions” button in Settings page, I am taken to <a href="https://github.com/mobxjs/mobx-react-lite/discussions/welcome_templates" rel="nofollow noopener">https://github.com/mobxjs/mobx-react-lite/discussions/welcome_templates</a> which shows error “Page not found”. Am I doing something wrong here?</p>

**Comment by evi-liu** [comment, marked as answer]
<p>Hi there! You encountered a bug that allowed you to setup Discussions without having enabled it on your repository yet.</p>
<p>If you’d like to enable Discussions, just let me know who the lead maintainers are, and I can help you get started.</p>
